src/overlays/actors/ovl_En_Fz/z_en_fz.c

```diff
@@ -180,56 +190,56 @@ void EnFz_Init(Actor* thisx, PlayState* play) {
     this->actor.velocity.y = 0.0f;
     this->actor.colChkInfo.mass = MASS_IMMOVABLE;
     this->actor.flags &= ~ACTOR_FLAG_ATTENTION_ENABLED;
-    this->unk_BC8 = 0;
-    this->unk_BCF = 0;
-    this->unk_BCC = 1;
-    this->unk_BCD = 0;
-    this->unk_BCE = 0;
-    this->unk_BD7 = 1;
-    this->unk_BD8 = 0;
+    this->unusedTimer = 0;
+    this->hitCounter = 0;
+    this->isBgEnabled = 1;
+    this->isMoving = false;
+    this->isColliderActive = false;
+    this->drawBody = true;
+    this->isDespawning = false;
```

Comment:
Shouldn't this be called `isDying` instead since you changed `EnFz_Despawn` to `EnFz_Die`?